usercp.php

```diff
@@ -4339,6 +4365,8 @@
 
 			if($subscriptions)
 			{
+				$latest_subscribed_threads = '';
+
```

Comment:
It is already initialized a few lines below in `feature` (https://github.com/mybb/mybb/commit/ed09961345ac1ed5f8d34774f995b4c430389602)